subworkflows/local/metagenomics_profiling.nf

```diff
@@ -21,6 +22,25 @@ workflow METAGENOMICS_PROFILING {
     ch_multiqc_files        = Channel.empty()
     ch_postprocessing_input = Channel.empty()
 
+    /*
+        UNTAR THE DATABASE IF NECESSARY
+    */
+
+    ch_database = ch_database
+    .branch{
+        untar: it ==~ /.*.tar.gz/
```

Comment:
Why not `it.extension`? Or something?

Reply:
I want to check for both tar and gzip, `it.extension` would only return `gz`, or not?